development/did-web-identifier

```diff
@@ -0,0 +1 @@
+{"did":"did:web:uncefact.github.io:project-vckit:test-and-development","alias":"uncefact.github.io:project-vckit:test-and-development","provider":"did:web","controllerKeyId":"bd3a7c525d86c18fa91f2a8f0354231b85aa0f7f269cc2a7914b6cca4087f0ae","keys":[{"kid":"bd3a7c525d86c18fa91f2a8f0354231b85aa0f7f269cc2a7914b6cca4087f0ae","kms":"local","type":"Ed25519","publicKeyHex":"bd3a7c525d86c18fa91f2a8f0354231b85aa0f7f269cc2a7914b6cca4087f0ae","privateKeyHex":"d10c5893361a0c15aa002387a6f608088dff9ef8e772307c5d06534fb391505b","meta":{"algorithms":["EdDSA","Ed25519"]}}]}
```

Comment:
is this the hardcode data? That means we only support Ed25519 (not for Secp256r1, Secp256k1, or X25519).

Reply:
@ldhyen99, what crypto suite must we use for enveloping proof (JOSE)?

Reply:
Answered below..